.github/workflows/MilestoneIssue.yml

```diff
@@ -0,0 +1,22 @@
+name: Milestone Issue
+
+on:
+  issues:
+    types: ["milestoned"]
+
+jobs:
+  create_card:
+    name: Create card
+    runs-on: ubuntu-latest
+
+    steps:
+      # https://github.com/actions/github-script
+      - uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const TODO_COLUMN = 4971951;
+            console.log("Creating Issue card");
+            // Action will fail if a card already exists for this issue
```

Comment:
It's too complicated to find a card with current API and there's no harm done => ROI would be bad now.

From online discussions, new API in alpha testing phase should provide better way of working with projects in the future.